frontend/packages/component-library/src/button/_baseButton.module.scss

```diff
@@ -37,7 +37,7 @@ a.button {
     }
   }
 
-  &:hover {
+  &:hover, &-hover  {
```

Comment:
(if we decide to implement this)

this might be simplified to smth like, `&:hover, &.button-withForcedHover`
From what I've seen previously when working on typography - it would be great to avoid using `&-classNameSuffix` classNames naming as it's not actually making styles easier to read and understand, but rather more complicated and not-straightforward.

That being said, this approach can still be used, but only for cases like in newest dsco components where it's used straight-forward (see https://github.com/code-dot-org/code-dot-org/blob/staging/frontend/packages/component-library/src/cms/section/section.module.scss)

Reply:
Fixed